api/src/pcapi/routes/backoffice/templates/accounts/get/general.html

```diff
@@ -199,6 +199,40 @@ <h5 class="modal-title">Revue manuelle</h5>
                 </div>
               </div>
             {% endif %}
+            {% if anonymize_form %}
+              <button class="btn btn-outline-primary lead fw-bold mt-2 justify-content-end"
+                      data-bs-toggle="modal"
+                      data-bs-target="#review-anonymization_confirmation"
+                      type="button">Anonymiser</button>
+              <div class="modal modal-lg fade"
+                   id="review-anonymization_confirmation"
+                   tabindex="-1"
+                   aria-labelledby="review-anonymization_confirmation-label"
+                   aria-hidden="true">
+                <div class="modal-dialog modal-dialog-centered">
+                  <div class="modal-content">
+                    <form action="{{ anonymize_public_accounts_dst }}"
+                          name="{{ anonymize_public_accounts_dst }}"
+                          method="post">
+                      <div class="modal-header">
+                        <h5 class="modal-title">anonymiser</h5>
```

Comment:
Au moins la majuscule, ici aussi...
```suggestion
                        <h5 class="modal-title">Anonymiser les données</h5>
```

Reply:
general ligne 218 
`                        <h5 class="modal-title">Anonymiser les données</h5>`
FAIT